docs/01-components-and-layout.md

```diff
@@ -91,6 +91,8 @@ Once the component is initialized it will render its markup. Replace the markup
 </div>
 ```
 
+Run the app by hitting `Ctrl-F5`. Now you should see a list of the specials available.
```

Comment:
I don't super love putting VS-specific directions in if we're not going to cover VS Code as well. Possible to add?

Reply:
I thought about that too, but, the majority of the screen shots involving the tooling also shows it, so I think we're safe. I can change this to "Run" as in most of the other places. Thanks.

Reply:
yeah that's a fair point.